### PR TITLE
added text about masking

### LINKED ISSUE #166: Discuss masking
A reviewer says:

> 4. Page 15. The approach of masking "low-recombination" portions of the chromosomes seems like an incomplete/indirect attempt to model the inherent limitations of sequencing to an "accessible" genome. 
> a. Shouldn't the approach instead be to drop "low complexity" regions (e.g., as defined by an excessive number of "N"'s in the reference, low mapability scores, or via tools like RepeatMasker?). This part of the pipeline seems open to refinement. 

We should discuss somewhere that other choices may be better, and say in the response that going deeper is beyond the scope of this paper.

This also should respond to the reviewer comments about whether masking is configurable (and add documentation for it if appropriate?).

### PR DESCRIPTION
**Pull request by jradrion**
I'm not sure that a discussion of masking within the discussion section is really warranted after rereading this reviewer's comment. The only mention of masking is in the methods as it is. I added further clarification to our explanation of masking in the methods, and I responded to the reviewer directly in the response document.

Let me know if you disagree or if you would like suggest any changes.

closes #166

### CODE REVIEW

**Comment by jradrion** [comment]
just noticed that "biases" on line 851 should be "biasing". I will correct this after I hear back from people.

**Comment by petrelharp** [comment]
Looks good to me. @andrewkern?

**Comment by gphocs-dev** [comment]
@jradrion, @petrelharp, I briefly reviewed the added text about masking and it looks fine. I do have a few suggestions for the response to reviewers:
1) The response is a bit long and contains some repetition. I think it could be tightened a bit.
2) "masking done using real data" --> "masking done on real genomic data"
3) "we will strongly consider allowing users to select from diverse masking approaches in future versions of the analysis pipeline". Aren't users already free to define their own masking? The response should clarify that masking is somewhat of an arbitrary choice and it can always be refined based on the user's preference.

File: popSimManu.tex [hunk 1]
```diff
@@ -847,7 +847,11 @@ \subsection*{Workflow for analysis of simulated data}
 used in empirical population genomic studies \citep[e.g.,][]{danecek20111000,langley2012genomic}.
 Specifically we masked all regions of \SI{1}{cM} or greater in the lowest 5th percentile of the empirical
 distribution of recombination, regions which are nearly uniformly absent for
-empirical analysis.
+empirical analysis. This approach to masking was chosen to prevent marginal trees with low or no recombination 
+from biases the comparisons of demographic inference methods. It should be noted that masking is not implemented
```
Comment (andrewkern) [inline popSimManu.tex]: typo here "biasing"

**Review by andrewkern — COMMENTED** [review]
one typo in here to fix @jradrion

**Comment by jradrion** [comment]
fixed the typo and added suggestions by @gphocs-dev

**Pull request merged**